aiverify-foundation/moonshot-ui
# Fix/ms 259 new attack module not listed

## Description

Code changes in this PR fixes the issues where newly added attack module and context strategy python files in moonshot-data, did not get listed in the Red Teaming flow.

## Motivation and Context


## Type of Change



 - fix: A bug fix 









## How to Test

Go through the Start Red Teaming session flow and verify that newly added attack module is listed in `set optional utilities` step.
- on `Set optional utilities` step, click back arrow
- add a new attack module python file to moonshot-data
- click next

On the Red Teaming session screen, verify new attack module is listed when Attack modules is clicked.
- while on Red Teaming session, add a new attack module python file to moonshot-data
- click Attack Modules to view the list and verify that newly added module is displayed and can be selected

Test context strategy listing in Red Teaming session using the same step as above

## Checklist

Please check all the boxes that apply to this pull request using "x":

- [ x]  I have tested the changes locally and verified that they work as expected.
- [x ]  I have followed the project's coding conventions and style guidelines.
- [x ]  I have rebased my branch onto the latest commit of the main branch.
- [x ]  I have performed a self-review of my own code.
- [x ]  I have read, understood and agree to the Developer Certificate of Origin below, which this project utilises.

## Screenshots (if applicable)

[If the changes involve visual modifications, include screenshots or GIFs that demonstrate the changes.]

## Additional Notes

[Add any additional information or context that might be relevant to reviewers.]

<details>
  <summary>Developer Certificate of Origin</summary>

 ```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
 ```
</details>
